src/components/Footer/Footer.tsx

```diff
@@ -32,21 +29,37 @@ const Text = styled.p`
   `}
 `
 
-const TextBold = styled.b`
-  ${({ theme }) => css`
-    color: ${theme.colors.black[100]};
-    border-bottom: 2px solid ${theme.colors.black[100]};
-  `}
+const TextButton = styled(TextButtonDefault)`
+  font-size: ${({ theme }) => theme.fonts.sizes.xmedium};
+  line-height: 23.44px;
 `
 
+const resetFocusToBody = () => {
```

Comment:
Remover e manter o padrao do browser.

Reply:
Fixed in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/43/commits/a4766ee9afc626bc1612c6c2796c84051a7e23f2)